test/lib/absinthe/extensions_test.exs

```diff
@@ -38,7 +38,7 @@ defmodule Absinthe.ExtensionsTest do
     pipeline =
       Schema
       |> Absinthe.Pipeline.for_document()
-      |> Absinthe.Pipeline.insert_after(Absinthe.Phase.Document.Result, MyPhase)
+      |> Absinthe.Pipeline.insert_before(Absinthe.Phase.Document.Result, MyPhase)
```

Comment:
Do you mind adding a second dedicated test case for this instead of changing this one? We want to test both directions really.

Reply:
sure